api/app/views/mno_enterprise/jpi/v1/impac/dashboards/_dashboard.json.jbuilder

```diff
@@ -1,4 +1,6 @@
-json.extract! dashboard, :id, :name, :full_name, :currency
+json.extract! dashboard, :id, :full_name, :currency
```

Comment:
Is `name` still used in some versions of impac! ? If so wouldn't hurt keeping it here